vllm/core/block/prefix_caching_block.py

```diff
@@ -658,6 +801,7 @@ def content_hash(self) -> Optional[int]:
         if prev_block_hash is None and not is_first_block:
             return None
 
+        assert len(self.token_ids) > 0
```

Comment:
nit: do we need this assert given `if not self.is_full`?

Reply:
You right, removed